### ISSUE TITLE
24.0.5 no compatible for this Nextcloud version

### ISSUE THREAD

**Issue by punkyard** [nextcloud/documentserver_community#269]
NC Version 24.0.4

A new version is available: Nextcloud 24.0.5
1 app has no compatible version for this Nextcloud version available.

Apps missing compatible version 
[Community Document Server ↗](https://apps.nextcloud.com/apps/documentserver_community)

I'd be very upset to loose this app which works perfectly with onlyoffice :/

**Comment by RoadrunnerBF** [comment]
✋  here the same - please Update Community Server to actual Version - my Team is so in love with the App ... Thank you ! :)

**Comment by christianlx** [comment]
Hello,
just to have some additional informations. I've updated to 24.0.6 and the application was not marked as no compatible anymore. Anyway, it does not work. Message "not available" when i try to edit file and error message "Bad healthcheck status" when I check it in CLI. The appinfo.xml is corrected with max version = 24 but it does not correct anything. Some of you have the same behaviour ?
Hope this is going to be corrected because for me it' a mean feature...

**Comment by mrbaseman** [comment]
I've just  updated my private Nextcloud installation to 24.0.6. My plan was to manually install documentserver_community 0.1.13 - but strangely, Onlyoffice 7.5.8  just continued working with the already installed document server 0.1.12. I maintain another installation, which is used by several other people, but I'm puzzled now. Shall I update that one, too?

**Comment by aaronhuggins** [comment]
@mrbaseman How do you have a version of onlyoffice (7.5.8) that's actually *higher* than the most recent release (7.2.1) as of October 20, 2022?

https://github.com/ONLYOFFICE/DocumentServer/releases

**Comment by aaronhuggins** [comment]
@mrbaseman nvm, I see. You're talking about the nextcloud app: https://github.com/ONLYOFFICE/onlyoffice-nextcloud/tags

Here's how everything breaks down:

- OnlyOffice Document Server v7.2.1 - Officially developed by Ascensio System SIA as a web-based document editor.
- onlyoffice v7.5.8 - Officially developed by the creator's of OnlyOffice as a connector for self-hosted OnlyOffice Document Server. Now at v7.6.8, but probably compatible(?) with any self-hosted document server version.
- documentserver_community v0.1.12 - Community developed unofficial bundle of the Document Server for self-hosting as a Nextcloud app. Currently bundles server version 6.4.2. The PR I opened bundles server version 7.2.1.

**Comment by mrbaseman** [comment]
Yes, it's the Onlyoffice version for Nextcloud 24 shown in the list of the Apps (https://apps.nextcloud.com/apps/onlyoffice) - the document server is the documentserver_community edition, still 0.1.12 and my plan was to update it to 0.1.13 to get Onlyoffice working again. But for some reason that I don't understand, the already installed combination still works - in contrast to other reports here. But maybe I just still don't understand how Onlyoffice, and the nextcloud App, and the document server interact.  Maybe your statement "...probably compatible(?) with any self-hosted document server version" gives a hint...

**Comment by aaronhuggins** [comment]
@mrbaseman Yeah, I can override version 0.1.12 on my installs but it was constantly getting disabled by other app updates and Nextcloud core updates. Additionally, I switched cloud providers and they refuse to support Collabora Office *at all* (dammit Cloudamo). That meant I was forced to use OnlyOffice, and no way in hell was I going to use an out-of-date version with potential security flaws.

**Comment by mrbaseman** [comment]
I'm trying v 0.1.13 - but I don't know how relevant a positive experience in my setup will be in the context of some comments above or in #260 that say the community document server wasn't compatible with nextcloud 24  -or even worse- not working at all if it was detected under some circumstances as being compatible

**Comment by aaronhuggins** [comment]
@mrbaseman Yeah, I noticed some behavioral issues with the plugin that mostly seemed due to upgrade and integrity checks which Nextcloud has implemented in core. I had to turn integrity checks off of my production system just to make the old app version install.

There's a rebuild task in the app and some autoconfig that seems partially broken. If the nextcloud folks will pay attention to my PRs or give me rights to the repo, I plan on tackling those two issues and getting this app working on Raspberry PI (if that's possible).
